src/common/interfaces/IVaultRegistryGatewayHandler.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.5.0;
+
+import {PoolId} from "../types/PoolId.sol";
+import {AssetId} from "../types/AssetId.sol";
+import {ShareClassId} from "../types/ShareClassId.sol";
+import {VaultUpdateKind} from "../libraries/MessageLib.sol";
+
+/// @notice Interface for VaultRegistry methods called by messages
+interface IVaultRegistryGatewayHandler {
```

Comment:
Nit: Curious why we didn't add this to the exisiting `src/common/interfaces/IGatewayHandlers`?

Reply:
Agree